data.tf

```diff
@@ -8,15 +8,14 @@ data "aws_ec2_instance_type" "this" {
 
 data "aws_ami" "ami" {
   most_recent = true
```

Comment:
Do you think we should perhaps add a var for specifying the owner account?
I'll take a look at other modules and how they allow users to specify an AMI.